src/System.Text.Json/src/System/Text/Json/Serialization/Policies/JsonValueConverterAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization.Policies
+{
+    [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class | AttributeTargets.Property, AllowMultiple = true)]
+#if MAKE_UNREVIEWED_APIS_INTERNAL
+    internal
+#else
+    public
+#endif
+    abstract class JsonValueConverterAttribute : Attribute
+    {
+        public Type PropertyType { get; protected set; }
+
+        public abstract JsonValueConverter<TValue> GetConverter<TValue>();
+
+        public virtual JsonValueConverter<object> GetConverter()
```

Comment:
I can not find anywhere in the code where this method is called. Can it be removed?

Reply:
I'll remove it. Thanks